### DISCUSSION TITLE
HTTP/2 server push support

### DISCUSSION THREAD

**Discussion by yatesco** [tokio-rs/axum#628] [Q&A] [answered]
Hi there - are there any plans to support the HTTP/2 server push protocol (the WS replacement, not the SMS replacement ;-): https://en.wikipedia.org/wiki/HTTP/2_Server_Push?

Thanks!

**Comment by davidpdrsn** [comment, marked as answer]
Not currently. Its unlikely that server push will go far [given Chrome is dropping support for it.](https://evertpot.com/http-2-push-is-dead) Because of this I haven't looked into it.

**Reply by yatesco** [reply]
thanks David
